src/test/java/uk/gov/hmcts/ccd/domain/service/caseaccess/CaseAccessOperationTest.java

```diff
@@ -84,67 +90,136 @@
 
     @Mock
     private RoleAssignmentService roleAssignmentService;
+
     @Mock
     private ApplicationParams applicationParams;
 
     @InjectMocks
     private uk.gov.hmcts.ccd.domain.service.caseaccess.CaseAccessOperation caseAccessOperation;
 
-
     @BeforeEach
     void setUp() {
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```

Comment:
Could also annotate class with `@ExtendWith(MockitoExtension.class)` to replace the need for setupMethod?

Reply:
Resolved in commit 96adc047a20c6d8d70f4b91d4d9b72e58c62b30e

Thanks for the pointer. I knew there was an annotation to do this but I took the easier route of just changing the deprecated call.  

I have now update this class (and some of the others already in this PR) to use `@ExtendWith(MockitoExtension.class)`: which had the additional side effect of:  

✅  allowing the removal of the nested calls to openMocks previously required to allow the `@Captor` to be correctly generated.     
✅ the additional mock usage verifications this introduced also flushed out a number of invalid mocks in some test cases.  These have all been corrected.